icechunk-python/benchmarks/most_recent.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+echo $(ls -t ./.benchmarks/**/* | head -n 1)
+pytest-benchmark compare --group=group,func,param --sort=fullname --columns=median --name=normal `ls -t ./.benchmarks/**/* | head -n 1`
```

Comment:
this is so dumb. but this is where we could upload to S3 for example. 

I confirmed with Coiled that their `--sync` option will not work for us.